### PR TITLE
Create node.js.yml

### PR DESCRIPTION
**Pull request by haikyuu**

### CODE REVIEW

File: src/utils.imba [hunk 1]
```diff
@@ -6,4 +6,4 @@ if import.meta.vitest
 	it "add", do
 		expect(add()).toBe 0
 		expect(add(1)).toBe 1
-		expect(add(1, 2, 3)).toBe 6
+		expect(add(1, 2, 3)).toBe 3
```
Comment (codeluggage) [inline src/utils.imba:9]: Looks so good here inline in GitHub!

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
```
Comment (codeluggage) [inline .github/workflows/node.js.yml:19]: Are we targeting v18 as a rule now?

**Review by codeluggage — COMMENTED** [review]
LGTM! Except for the intentionally failing test ofc :)

File: .github/workflows/node.js.yml (hunk 2 above)
Comment (haikyuu) [reply .github/workflows/node.js.yml:19]: 16 (LTS) is the recommended version now, so we'll target it

**Pull request merged**